src/main/java/Main.java

```diff
@@ -1,6 +1,49 @@
+import java.util.Scanner;
 
 public class Main {
+    private final static int automobileCount = 3;
+    private final static int maxSpeed = 250;
```

Comment:
В Java и Kotlin константы принято называть в соответствии со SCREAMING_SNAKE_CASE, т.е. здесь лучше переименовать в:
```suggestion
    private final static int AUTOMOBILE_COUNT= 3;
    private final static int MAX_SPEED = 250;
```